acra/acra-in-depth/cryptography-and-key-management/_index.md

```diff
@@ -13,7 +13,7 @@ Cryptography is widely used across all Acra services for:
 * transport protection and authentication: during mutual authentication and encryption used in 
   [Themis Secure Session](/themis/crypto-theory/cryptosystems/secure-session/) and [TLS](/acra/configuring-maintaining/tls/);
 * [audit logging](/acra/security-controls/security-logging-and-events/audit-logging/): when calculating integrity checks of log messages and log chains;
-* [password hashing](/acra/configuring-maintaining/general-configuration/acra-authmanager#auth-file) of registered users of Acra Web Configuration UI;
+* [password hashing](/acra/configuring-maintaining/general-configuration/acra-authmanager#auth-file) of registered users of Acra Web Configuration UI (deprecated and removed after 0.90.0);
```

Comment:
```suggestion
* [password hashing](/acra/configuring-maintaining/general-configuration/acra-authmanager#auth-file) of registered users of Acra Web Configuration UI (deprecated and removed since 0.91.0);
```